packages/certs-v5/test/commands/certs/auto/enable.js

```diff
@@ -93,7 +91,6 @@ foo.example.org  CNAME        foo.example.org.herokudns.com
 
     return certs.run({
         app: 'example',
-        args: ['--wait'],
```

Comment:
Sorry to not mention earlier, but it will be nice to keep at least one of this --wait to test that the flag is there and works

Reply:
We have tests that check that on the "[enable.--wait.js](https://github.com/heroku/cli/blob/7f92af3027caa02ad4b1c5b5b41d77d6deebf95d/packages/certs-v5/test/commands/certs/auto/enable.--wait.js)" file. I've decided to split this in two files because it was getting too large.